runtime/oti/j9nonbuilder.h

```diff
@@ -3007,7 +3007,11 @@ typedef struct J9Class {
 	UDATA castClassCache;
 	void** jniIDs;
 	UDATA lockOffset;
-	UDATA lockReservationHistory;
+#if defined(J9VM_ENV_DATA64)
+	U_32 paddingForGLRCounters; // This is used to preserve alignment under 64 bit.
```

Comment:
You can't use C++-style comments in this header file - it is also used by C code (and some compilers are pedantic).
Same goes for second `paddingForGLRCounters` declaration below.

Reply:
Okay, I'll change those.